charts/px-central/values.yaml

```diff
@@ -196,7 +196,7 @@ images:
     registry: docker.io
     repo: portworx
     imageName: mysql
-    tag: 5.7.44
+    tag: 8.0.33
```

Comment:
Please push the new image to docker before merging this PR.

Reply:
Same change needed in versions.yaml as well I hope.

Reply:
done